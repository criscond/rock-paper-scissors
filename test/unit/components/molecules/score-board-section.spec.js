import scoreBoardSection from '../../../../src/js/components/molecules/score-board-section';
describe('Score Board Section', () => {

	it('should create a section', () => {
		expect(scoreBoardSection()).toMatchSnapshot();
	});
});
