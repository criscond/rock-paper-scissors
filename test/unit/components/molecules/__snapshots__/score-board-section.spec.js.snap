// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score Board Section should create a section 1`] = `
<section
  class="column"
>
  <span
    class="score"
    id="score"
  >
    -= Score Board =-
  </span>
</section>
`;
